curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-destructuring-assignment-to-assign-variables-from-nested-objects.english.md

```diff
@@ -9,7 +9,7 @@ challengeType: 1
 We can similarly destructure <em>nested</em> objects into variables.
 Consider the following code:
 <blockquote>const a = {<br>&nbsp;&nbsp;start: { x: 5, y: 6},<br>&nbsp;&nbsp;end: { x: 6, y: -9 }<br>};<br>const { start : { x: startX, y: startY }} = a;<br>console.log(startX, startY); // 5, 6</blockquote>
-In the example above, the variable <code>start</code> is assigned the value of <code>a.start</code>, which is also an object.
+In the example above, the variable <code>startX</code> is assigned the value of <code>a.start.x</code>, which is also an object.
```

Comment:
Thank you for pointing this out in your PR.  You would also need to replace `, which is also an object.` with a period to just end that sentence, because `a.startX.x` is not an object.

Reply:
@RandellDawson yes, a.start.x is not an object but a number. I have made the change